pkg/app/instances/docker.go

```diff
@@ -43,6 +43,9 @@ type DockerInstanceManager struct {
 	Client client.Client
 }
 
+const CreateHostOPType string = "createhost"
```

Comment:
let's use a type alias here and a `const` block

```
type OpType string

const (
	CreateHostOpType OpType = "createhost"
	DeleteHostOPType OpType = "deletehost"
)
```

Reply:
Defined `OPType`.